ui/src/selection-panel/selection-panel-geometry-section.tsx

```diff
@@ -137,7 +131,7 @@ const SelectionPanelGeometrySection: React.FC<GeometryPlansPanelProps> = ({
     );
 
     React.useEffect(
-        () => [...fetchedPlans.keys()].forEach(fetchPlanLayout),
+        () => void fetchPlanLayouts([...fetchedPlans.keys()]),
```

Comment:
Tämä `void` tässä on vähän jännä 🤔 Sen varmaan voisi vaan poistaa?

Reply:
Voisi korvata blokilla, se olisi ehkä selkeämpi ja syntaksissa kelpaa tähän kohtaan, mutta jotainhan siinä pitää olla: Tuo on useEffect()-kutsu, eli jos siitä tulisi palautusarvo, React yrittäisi ajaa sitä destruktorina.

Reply:
Toinen vaihtoehto taitaisi olla tehdä tuosta blokki. Jos tuon kirjoittaisi `{ fetchPlanLayouts[...] }`, niin se ei yrittäisi myöskään palauttaa mitään ja olisi (ainakin minulle) tutumpi syntaksi. Kolmas vaihtoehto olisi tehdä ihan oma funktionsa tuolle, joka ei vaan sisäisesti palauta mitään. Se voisi itse asiassa olla selkein veto, sillä uuden funktion nimeen voisi suoraan dokumentoida miksi näin halutaan tehdä, koska se ei välttämättä ole myöhemmälle lukijalle suoraan ilmeistä. Muussa tapauksessa jättäisin koodikommentin indikoimaan miksi halutaan vaan hakea muttei palauttaa